### DISCUSSION TITLE
Animixplay has shut down forever. What do we plan to do for the sources and scraper?

### DISCUSSION THREAD

**Discussion by snipeart007** [pystardust/ani-cli#957] [General] [answered]
Maintainers: @CoolnsX, @justchokingaround, @Derisis13 

Guys, it's bad but true. Animixplay has been shut off by the developers forever. They messed up their scraper backend, and are now out of motivation to fix it.

What should be the new reliable source?

I saw V4 branch where allanime has been added as a new source.
But it's still incomplete as it yet to be perfected.

And just for curiosity,
What is the new external launcher stuff in the V4 branch?

**Comment by port19x** [comment, marked as answer]
We're currently doing a full rewrite of ani-cli to get rid of some old / unused code.
Allanime is the current goal for the source.
Since we're making fzf our standard UI driver we can easily enable support for alternative launchers.
Previously we used a hacky interface that ran directly in the terminal

**Reply by justchokingaround** [reply]
yes, basically the current external launcher config allows for user to use fzf, rofi, wofi, tofi etc. whatever they want. why use a cli to watch anime, if you can watch it without even opening a terminal 😎

**Reply by port19x** [reply]
@justchokingaround had to unselect myself because of a known bug with the galaxy brain achievement.
Can you remark me?
